packages/python-pulp-ansible/python-pulp-ansible.spec

```diff
@@ -50,6 +50,12 @@ rm -rf %{pypi_name}.egg-info
 %{python3_sitelib}/pulp_ansible-%{version}-py%{python3_version}.egg-info
 
 %changelog
+* Thu Oct 01 2020 Evgeni Golov - 1:0.4.1-1
+- Release python-pulp-ansible 0.4.1
+
+* Mon Sep 28 2020 Evgeni Golov 1:0.4.0-1
+- Update to 0.4.0
+
```

Comment:
I wonder how useful this entry is

Reply:
It's not, just tools doing tools jobs ;)